tb_pulumi/__init__.py

```diff
@@ -0,0 +1,182 @@
+'''Standardization library for the usage of Pulumi in Python at Thunderbird. For an overview of how
+to use this library, read the README: https://github.com/thunderbird/pulumi/blob/main/README.md
+'''
+
+import boto3
+import pulumi
+import yaml
+
+from datetime import date
+from os import environ, getlogin
+from socket import gethostname
+
+
+# Internalize runtime information
+AWS_CLIENTS = {}
+AWS_SESSION = boto3.session.Session()
+AWS_ACCOUNT_ID = None
+AWS_REGION = None
+PROJECT = pulumi.get_project()
+PULUMI_CONFIG = pulumi.Config()
+STACK = pulumi.get_stack()
+
+# Make certain common data accessible through this module
+ASSUME_ROLE_POLICY={
+    'Version': '2012-10-17',
+    'Statement': [{
+        'Sid': '',
+        'Effect': 'Allow',
+        'Principal': {
+            'Service': None
+        },
+        'Action': 'sts:AssumeRole'
+    }]
+}
+COMMON_TAGS = {
+    'project': PROJECT,
+    'pulumi_last_run_date': str(date.today()),
```

Comment:
I ended up removing it entirely because it leads to every single run producing 30+ items to update, which convolutes the process of making small changes. Happy to add this back in if there is consensus. We will be able to tell the last run time by using the `pulumi state history` command. Andrei also mentioned switching to using Pulumi Cloud as a state backend instead of S3. If/when we do that, the history will then be exposed through a handy and very clear web UI.